mir-ci/mir_ci/pytest.ini

```diff
@@ -13,3 +13,6 @@ asyncio_mode=auto
 log_cli = True
 log_format = %(asctime)s %(levelname)s %(message)s
 log_date_format = %Y-%m-%d %H:%M:%S
+filterwarnings =
+    error
+    ignore::UserWarning
```

Comment:
I am now testing that `with pytest.raises(Exception):`, right?

Reply:
> I am now testing that `with pytest.raises(Exception):`, right?

I'd go for a `UserWarning`, or whatever's appropriate here, otherwise you might catch an exception you didn't, actually, expect :)

Reply:
Done!

Reply:
Fixing CI... one sec...